query/control/controller.go

```diff
@@ -212,24 +210,15 @@ func New(config Config) (*Controller, error) {
 	return ctrl, nil
 }
 
-// Query satisfies the AsyncQueryService while ensuring the request is propagated on the context.
-func (c *Controller) Query(ctx context.Context, req *query.Request) (flux.Query, error) {
+func (c *Controller) Query(ctx context.Context, compiler flux.Compiler) (flux.Query, error) {
```

Comment:
Why get rid of `query.Request`? It doesn't look like it was very useful, is this just code cleanup?

Reply:
`query.Request` is a 2.x type. 1.x has `QueryRequest` but that doesn't have the same methods / contents.